packages/docs/src/components/navigator.tsx

```diff
@@ -112,7 +112,13 @@ export class Navigator extends React.Component<INavigatorProps, INavigatorState>
     }
 
     public componentDidMount() {
-        this.sections = flattenSections(this.props.pages);
+        this.sections = [];
+        eachLayoutNode(this.props.items, (node, parents) => {
+            const { reference, title } = node;
+            const path = parents.map((p) => p.title).reverse();
+            const filterKey = [...path, title].join("/");
```

Comment:
nope, see the next line where `path` is used in the object and should not contain title.